lib/puppet/tab.ts

```diff
@@ -101,9 +115,8 @@ export default class Tab implements TabActor {
     return false;
   }
 
-  async hideElements(selectors: string[], frameId = 0) {
-    // TODO implement this
-    return Promise.resolve(true)
+  async hideElements(selectors: string[], frameId = 0, method: HideMethod = 'display') {
+    return await this.frames[frameId].evaluate(`window.autoconsentTestUtils.hideElementsUtil(${JSON.stringify(selectors)}, '${method}')`);
```

Comment:
The idea is to have reusable functions that are called from the content script, and injected by the playwright harness. I think the other RPC commands can be refactored in the same way, to make test setup closer to the production version.

Since those functions may call into each other, they need to be either saved in globals, or injected in `evaluate()` call. I figured that  The edge case with navigation is a good point, so I guess we should inject them every time indeed, as long as we keep the background page model.

As for the function arguments, I tried passing them at first, but in this case the first argument has to be a function, otherwise Playwright always returns undefined as the result. If we inject multiple functions, their source has to be rendered in a single string, I don't think there's a way around that.

I'll poke around a bit more and see if I can get rid of `_initPlaywrightUtils()`